suites/quincy/cephadm/tier-1_os_tuning_profiles.yaml

```diff
@@ -0,0 +1,330 @@
+            - service_type: prometheus
+              placement:
+                count: 1
+                nodes:
+                  - node1
+            - service_type: grafana
+              placement:
+                nodes:
+                  - node1
+            - service_type: alertmanager
+              placement:
+                count: 2
+                label: alertmanager
+            - service_type: node-exporter
+              placement:
+                host_pattern: "*"
+            - service_type: crash
+              placement:
+                host_pattern: "*"
+        - config:
+            command: shell
+            args:                 # sleep to get all services deployed
+              - sleep
+              - "300"
+  - test:
+      name: Service deployment with spec
+      desc: Add services using spec file.
+      module: test_cephadm.py
+      polarion-id: CEPH-83573746
+      config:
+        steps:
+        - config:
+            command: apply_spec
+            service: orch
+            specs:
+            - service_type: osd
+              service_id: all-available-devices
+              placement:
+                host_pattern: "*"
+              spec:
+                data_devices:
+                  all: "true"                         # boolean as string
+                encrypted: "true"                     # boolean as string
+        - config:
+            command: shell
+            args:                 # sleep to get all services deployed
+              - sleep
+              - "300"
+  - test:
+      name: MDS Service deployment with spec
+      desc: Add MDS services using spec file
+      module: test_cephadm.py
+      polarion-id: CEPH-83574728
+      config:
+        steps:
+          - config:
+              command: shell
+              args:               # arguments to ceph orch
+                - ceph
+                - fs
+                - volume
+                - create
+                - cephfs
+          - config:
+              command: apply_spec
+              service: orch
+              specs:
+                - service_type: mds
+                  service_id: cephfs
+                  placement:
+                    label: mds
+          - config:
+              command: shell
+              args:              # sleep to get all services deployed
+                - sleep
+                - "120"
+  - test:
+      name: NFS Service deployment with spec
+      desc: Add NFS services using spec file
+      module: test_cephadm.py
+      polarion-id: CEPH-83574729
+      config:
+        steps:
+          - config:
+              command: apply_spec
+              service: orch
+              validate-spec-services: true
+              specs:
+                - service_type: nfs
+                  service_id: nfs-rgw-service
+                  placement:
+                    nodes:
+                      - node4
+                  spec:
+                    port: 2049
+  - test:
```

Comment:
Hi Mohit, 
One doubt here, these tests are already part of cephci/suites/quincy/cephadm/tier-1_service_apply_spec.yaml right? So in that case, can we skip those here (atleast few) and then use the tuning related tests alone here?
Please correct me if I got it wrong here.

Reply:
Hi @pranavprakash20 

As per the test case we need to configure all the services before applying tune profile, So we kept all the services.